distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -110,6 +110,12 @@ type Column
         mapping = self.connection.dialect.get_type_mapping
         mapping.sql_type_to_value_type self.sql_type
 
+    ## PRIVATE
+       Returns the `Value_Type` based on the data in the column.
+    resolved_value_type : Value_Type
+    resolved_value_type self =
+        self.value_type
```

Comment:
```suggestion
    ## PRIVATE
       Returns the `Value_Type` based on the actual data in the column.
       
       For typed columns, this is the same as `value_type`. Only for `Mixed` type columns this will attempt to find a more specific type.
    inferred_value_type : Value_Type
    inferred_value_type self =
        self.value_type
```

Personal preference but I like the name `inferred` more than `resolved`. Resolve sounds like resolving some type using some rules for converting the types. IMHO infer sounds more like "I looked at the types and found the mst specific one".

I also suggest a clarification in the comment to make it clearer what this function does and what is its usage.